pysatNASA/instruments/ace_epam_l2.py

```diff
@@ -0,0 +1,103 @@
+# -*- coding: utf-8 -*-
+"""Module for the Advanced Composition Explorer (ACE) EPAM instrument.
+
+Properties
+----------
+platform
+    'ace'
+name
+    'epam_l2'
+tag
+    'base' or 'key'
+inst_id
+    '12sec', '5min', '1hr'
+
+References
+----------
+- Stone, E., Frandsen, A., Mewaldt, R. et al. The Advanced Composition Explorer.
+  Space Science Reviews 86, 1–22 (1998). https://doi.org/10.1023/A:1005082526237
+- Gold, R., Krimigis, S., Hawkins, S. et al. Electron, Proton, and Alpha Monitor
+  on the Advanced Composition Explorer spacecraft. Space Science Reviews 86,
+  541–562 (1998). https://doi.org/10.1023/A:1005088115759
+
+Note
+----
+- Level 1 ACE data is maintained at pysatSpaceWeather.
+- Release notes at
+  https://cdaweb.gsfc.nasa.gov/pub/data/ace/epam/epam_level2_release_notes.txt
+
+Warnings
+--------
+- The cleaning parameters for the instrument are still under development.
+
+"""
+
+import datetime as dt
+import functools
+
+from pysat.instruments.methods import general as mm_gen
+
+from pysatNASA.instruments.methods import ace as mm_ace
+from pysatNASA.instruments.methods import cdaweb as cdw
+from pysatNASA.instruments.methods import general as mm_nasa
+
+# ----------------------------------------------------------------------------
+# Instrument attributes
+
+platform = 'ace'
+name = 'epam_l2'
+tags = {'base': 'ACE/EPAM Solar Energetic Particle Base Data',
+        'key': 'ACE/EPAM Solar Energetic Particle Key Parameters'}
+inst_ids = {'12sec': ['base'],
+            '5min': ['key', 'base'],
+            '1hr': ['key', 'base']}
+pandas_format = False
+
+# ----------------------------------------------------------------------------
+# Instrument test attributes
+
+_test_dates = {id: {tag: dt.datetime(2022, 1, 1) for tag in inst_ids[id]}
+               for id in inst_ids.keys()}
+
+# ----------------------------------------------------------------------------
+# Instrument methods
+
+
+# Use standard init routine
+init = functools.partial(mm_nasa.init, module=mm_ace, name=name)
+
+# No cleaning, use standard warning function instead
+clean = mm_nasa.clean_warn
```

Comment:
Recommend using the clean function to replace custom fill values with np.nan.

Reply:
Ok, now the meta looks better, but the fill values are still awful.  How about using preprocess to update them to np.nan?

```
units                                                         #/cc
long_name                                                H_Density
notes            Np is the proton number density in units of cm...
desc                      Solar Wind Proton Number Density, scalar
value_min                                                      0.0
value_max                                                    200.0
fill                            -9999999848243207295109594873856.0
plot_label                                               H_Density
axis                                                              
scale                                                       linear
FIELDNAM                                        Proton No. density
DICT_KEY                                            density>number
standard_name                                   Proton No. density
value_min0                                                    -1.0
value_min1                                                    -1.0
value_min2                                                    -1.0
value_max0                                                    -1.0
value_max1                                                    -1.0
value_max2                                                    -1.0
SCALEMIN0                                                     -1.0
SCALEMIN1                                                     -1.0
SCALEMIN2                                                     -1.0
SCALEMAX0                                                     -1.0
SCALEMAX1                                                     -1.0
SCALEMAX2                                                     -1.0
FORM_PTR                                                          
LABL_PTR_1                                                        
UNIT_PTR                                                          
SCALETYP                                                          
SCALEMIN                                                       0.0
SCALEMAX                                                     100.0
AVG_TYPE                                                          
children                                                      None
```

Reply:
`preprocess` or `clean`?  I'm kind of leaning toward `clean` since that gives users the option to use the existing values.  This also matches the choice for other modules here (eg, `cnofs_plp`).

Reply:
I lean towards whatever makes the most sense for pysatNASA, so `clean` sounds great!

Reply:
pysat is not letting me update the fill value in meta to nan :/

Reply:
Can you send me a code snippet/the instrument mod file that isn't working via email or slack?